### DISCUSSION TITLE
Feature request: Add wheels for Apple Sillicon

### DISCUSSION THREAD

**Discussion by Thijss** [networkit/networkit#1174] [Ideas]
Current there are only wheels for MacOS Intel chips. Would be nice to add wheels for M1/M2 chips as well.

https://github.com/pypa/cibuildwheel seems to support this without issues

**Comment by fabratu** [comment]
Hi Thijss,

thanks for indirectly reminding me to upload arm-wheels for 10.1. They are now on PyPi for Python 3.10 and 3.11.

Unfortunately, the arm/universal-builds in cibuildwheel didn't work in the past due to incompatibility problems with the underlying OpenMP/libomp. Maybe that now works. There is still commented code in `build_wheels.yml`[1] (likely needs to be updated).

If you are familiar with the topic - feel free to post a PR adding the missing feature.

[1] https://github.com/networkit/networkit/blob/master/.github/workflows/build_wheels.yml#L130
